src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -54,28 +54,18 @@
 examples:
   - name: Update a subscription lock with new notes and type
     text: >
-        az account lock update --name lockName --notes newNotesHere --lock-type CanNotDelete
+        az account lock update --name MyLockName --notes newNotesHere --lock-type CanNotDelete
 """
 
 helps['account management-group'] = """
 type: group
-short-summary: Manage Azure Management Groups.
+short-summary: Manage Azure subscription-level management groups.
+long-summary: For more information on Azure management groups, see [What are Azure management groups?](https://learn.microsoft.com/en-us/azure/governance/management-groups/overview)
 """
 
 helps['account management-group create'] = """
 type: command
 short-summary: Create a new management group.
-long-summary: Create a new management group.
```

Comment:
@zhoxing-ms, as part of general file cleanup, parameter information was removed under the assumptions 1.) this information is not in use here and 2.) parameter information is now in _params.py.  Is this correct?

Reply:
In fact, the help message of the CLI has four layers, the priority of the help message written in `_help.py` is the highest. For more information, please refer to this doc: [Azure CLI Help System](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_help.md#help-layers). Since there is no repeated help message declared in [_params.py](https://github.com/Azure/azure-cli/blob/22c9d89df193b659179266c405bc713723f74bb5/src/azure-cli/azure/cli/command_modules/resource/_params.py#L594-L596
), so the information here is useful. 

May I ask why you want to remove these help messages?

Reply:
@zhoxing-ms , I am working under the assumption that 1.) this information is not in use here and 2.) parameter information is now in _params.py. Is this correct?

Reply:
> Since there is no repeated help message declared in [_params.py](https://github.com/Azure/azure-cli/blob/22c9d89df193b659179266c405bc713723f74bb5/src/azure-cli/azure/cli/command_modules/resource/_params.py#L594-L596), so the information here is useful.

No, as I mentioned earlier, the current help message comes from `_help.py`.